nomos-services/data-availability/verifier/src/lib.rs

```diff
@@ -98,9 +100,8 @@ where
 
 impl<Backend, N, S> ServiceData for DaVerifierService<Backend, N, S>
 where
-    Backend: VerifierBackend,
+    Backend: VerifierBackend<Error = DynError>,
     Backend::Settings: Clone,
-    Backend::Error: Error,
```

Comment:
Why this error changes? We are going to a more specific type. Is the generic not enough? I think expanding the bound of the generic to what's needed would have been better.

Reply:
To keep the error generic I've created a new verifier backend error. My idea was that DynError is generic enough for the service.